test/extended/router/router.go

```diff
@@ -129,7 +129,11 @@ func waitForNamedRouterServiceIP(oc *exutil.CLI, name string) (string, error) {
 			}
 		}
 		o.Expect(err).NotTo(o.HaveOccurred())
-		host = svc.Spec.ClusterIP
+		if len(svc.Status.LoadBalancer.Ingress) > 0 && len(svc.Status.LoadBalancer.Ingress[0].Hostname) > 0 {
```

Comment:
Hmm, not sure if `Status.LoadBalancer` will exist right away... would it be more stable to switch on `Spec.Type`? e.g. if `Type=Cluster` use ClusterIP, if `Type=LoadBalancer` use Status.LoadBalancer

Reply:
ok will switch it based on type.